inc/documenttype.class.php

```diff
@@ -162,15 +162,29 @@ static function getSpecificValueToSelect($field, $name='', $values='', array $op
    /**
     * @since version 0.85
    **/
-   static function showAvailableTypesLink() {
+   static function showAvailableTypesLink($options = []) {
```

Comment:
Parameter not documented (missing function doc as well). Add `@since` for the param